client/src/components/ProfilePicture/ProfilePicture.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./ProfilePicture.css";
+import { Link } from "react-router-dom";
+
+// Function based React Component
+const ProfilePicture = props => {
+  // Default Class to apply to Component
+  let classList = "ProfilePicture";
+  let types = ["extra-small", "small", "medium", "large"];
+  if (types.includes(props.type)) {
+    classList += ` ${props.type}`;
+  }
```

Comment:
If you set a default value for `type`, then you wont need to do this. Can you look up `defaultProps` in React and set a default? Then you can just to `ProfilePicture ProfilePicture--${type}` for the `className` attribute